components/brave_rewards/browser/extension_rewards_service_observer.cc

```diff
@@ -266,4 +267,39 @@ void ExtensionRewardsServiceObserver::OnPendingContributionSaved(
   event_router->BroadcastEvent(std::move(event));
 }
 
+void ExtensionRewardsServiceObserver::OnPublisherListNormalized(
+    RewardsService* rewards_service,
+    brave_rewards::ContentSiteList list) {
+  auto* event_router = extensions::EventRouter::Get(profile_);
+  if (!event_router) {
+    return;
+  }
+
+  std::vector<extensions::api::brave_rewards::OnPublisherListNormalized::
+        PublishersType> publishers;
+
+  for (size_t i = 0; i < list.size(); i ++) {
+    publishers.push_back(
+        extensions::api::brave_rewards::OnPublisherListNormalized::
+        PublishersType());
+
+    auto& publisher = publishers[publishers.size() -1];
```

Comment:
It looks like we're assigning the same element in `publishers` every iteration. Want to make sure this is intended.

Reply:
Missed the line above. disregard.